packages/nuget-exe/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "nuget-exe",
+  "version": "4.9.1",
```

Comment:
The patch version is intentionally off by one since Beachball will bump this version before publishing.